data/seeds/03_videos.js

```diff
@@ -5,46 +5,133 @@ exports.seed = function (knex, Promise) {
       return knex("videos").insert([
         {
           video_title: "Testing",
-          location: "https://www.youtube.com/watch?v=82Q6DRqf9H4",
+          location:
+            "https://elasticbeanstalk-us-east-1-427400502172.s3.amazonaws.com/Videosforbackend/mayhem--push_every_button-testing--82Q6DRqf9H4.mp4",
```

Comment:
Here is the path to the actual DS files: 
https://artificial-artist.s3.amazonaws.com/

Here are the file names:
![image](https://user-images.githubusercontent.com/740373/83484370-9c53be80-a459-11ea-94e1-563e2d66ee4a.png)

Reply:
Excellent, Will get these in.

Reply:
revisits the seed file to include the correct S3 bucket and Actual created videos here https://github.com/Lambda-School-Labs/artificial-artist-be/pull/23/commits/ed692ff7bdc86e087bf5959642bb319b8a814096